web/models/user.ex

```diff
@@ -0,0 +1,29 @@
+defmodule MovieWagerApi.User do
+  use MovieWagerApi.Web, :model
+
+  schema "users" do
+    field :screen_name, :string
+    field :profile_image_url, :string
+    field :twitter_id, :integer
+    field :name, :string
```

Comment:
That makes sense if Twitter is the only way that people will ever log in and you don't intend to support a) actual accounts, or b) alternative signon accounts like Facebook. Not sure if you're intending to expand it outward.

I'm also not crazy about our naming conventions suffering because theirs are stupid, but I do get the benefit of consistency.